test/functions.test.js

```diff
@@ -1,24 +1,192 @@
 // IMPORT MODULES under test here:
 import { 
-    myFunction,
-} from '../functions.js';
+    addExclamationPoints, 
+    multiplyBySeven, 
+    multiplyBy12ThenHalve,
+    divideThenMultiply,
+    returnAsAnArray,
+    returnAsAString,
+    makeLuckyGreeting,
+    getSecondItem,
+    getLastItem,
+    getRandomNumber } from '../functions.js';
 
 const { test, skip } = QUnit;
 
-test('this test should pass', (expect) => {
-    const expected = true;
+test('addExclamationPoint should return a string with 3 exclamation points added to the end', (expect) => {
+    const expected1 = 'puppy!!!';
+    const actual1 = addExclamationPoints ('puppy');
 
-    const actual = myFunction();
+    expect.equal(actual1, expected1, 'puppy!!!');
 
-    expect.equal(actual, expected);
+    const expected2 = 'guppy!!!';
+    const actual2 = addExclamationPoints ('guppy');
+
+    expect.equal(actual2, expected2, 'guppy!!!');
+
+    const expected3 = 'dopey!!!';
+    const actual3 = addExclamationPoints ('dopey');
+    expect.equal(actual3, expected3, 'dopey!!!');
+    
+});
+
+test('multiplyBySeven should return a number by taking in a number and multiply it by seven', (expect) => {
+    const expected1 = 21;
+    const actual1 = multiplyBySeven (3);
+
+    expect.equal(actual1, expected1, 'the return should be 21');
+
+    const expected2 = 70;
+    const actual2 = multiplyBySeven (10);
+
+    expect.equal(actual2, expected2, 'the return should be 70');
+
+    const expected3 = -21;
+    const actual3 = multiplyBySeven (-3);
+
+    expect.equal(actual3, expected3, 'the return should be -21');
```

Comment:
Nice job accounting for negative numbers.  Testing for edge cases is an important part of writing reliable tests.